### ISSUE TITLE
Running under other N way K shot settings 

### ISSUE THREAD

**Issue by raspberryice** [chen700564/causalFSED#1]
Hi, 
Which parts of the code do I need to change to run the model under other N-way K-shot settings?

**Comment by chen700564** [comment]
There is a k parameter in main.py. You can run ''python main.py -k K .... '' for K-shot. 
For N-way, the few-shot setting in this code is that N is the number of types in test set. You can change the reader.py to implement traditional N-way setting.

**Comment by raspberryice** [comment]
Thanks for the quick reply! So for ACE you are doing 10+1 way 5 shot classification? But then in Figure 2 you say that you compare with the episode setting for 5+1 way 5 shot classification? This confuses me a bit.

**Comment by chen700564** [comment]
Yes, but we compare 5+1way5shot under episode with under ambiguty setting . Our setting is not compared with the result of them directly.
